guides/common/modules/proc_migrating-project-or-smartproxy-using-clone.adoc

```diff
@@ -0,0 +1,22 @@
+[id="migrating-{project-context}-or-proxy-using-clone_{context}"]
+= Migrating {Project} or {SmartProxy} to {EL-abbr}{nbsp}9 using clone
+
+You can clone your existing {ProjectServer} and {SmartProxyServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+ifdef::satellite[]
+Create a backup of the existing {ProjectServer} and {SmartProxyServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You can use cloning for {ProjectServer} backups but not for {SmartProxyServer} backups.
+====
+endif::[]
+
+.Procedure
+. Perform a full backup of your {ProjectServer} or {SmartProxy}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin#Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer} or {SmartProxyServer}] in _{AdministeringDocTitle}_.
```

Comment:
This looks like a broken link (it doesn't open any specific section).

Reply:
It should ideally point to 
https://docs.redhat.com/it/documentation/red_hat_satellite/6.16/html/administering_red_hat_satellite/backing-up-satellite-server-and-capsule_admin#Performing_a_Full_Backup_admin